media/static-bundles.json

```diff
@@ -1019,6 +1025,17 @@
       ],
       "name": "firefox_whatsnew_67"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/base/mozilla-fxa.js",
+        "js/base/mozilla-fxa-init.js",
+        "js/base/mozilla-fxa-form.js",
+        "js/base/mozilla-monitor-button.js",
+        "protocol/js/protocol-modal.js"
+      ],
+      "name": "firefox_whatsnew_trailhead"
```

Comment:
Nit: I like to name the CSS and JS bundles the same thing unless we think they're going to be used separately.